commercetools-models/src/main/java/io/sphere/sdk/products/search/PriceSelectionBuilder.java

```diff
@@ -80,6 +89,22 @@ public PriceSelectionBuilder priceChannel(@Nullable final Referenceable<Channel>
         return this;
     }
 
+    public PriceSelectionBuilder storeProjection(@Nullable final String storeProjection) {
+        this.storeProjection = storeProjection;
+        return this;
+    }
+
+    public PriceSelectionBuilder localeProjection(@Nullable final List<String> localeProjection) {
+        this.localeProjection = localeProjection;
+        return this;
+    }
+
+    public PriceSelectionBuilder plusLocaleProjection(@Nullable final String localeProjection) {
+        this.localeProjection = emptyList();
```

Comment:
Here the code should be:
```java
if (localeProjection == null {
   this.localeProjection = new ArrayList<>();
}
```
And please: never use `emptyList()` if you need a mutable list (as in this case). See also https://www.baeldung.com/java-collections-emptylist-new-list

Reply:
thanks for the explanation,  I'm gonna fix it!